av/sidedata/sidedata.pyx

```diff
@@ -25,6 +25,8 @@ Type = define_enum('Type', __name__, (
     ('SPHERICAL', lib.AV_FRAME_DATA_SPHERICAL),
     ('CONTENT_LIGHT_LEVEL', lib.AV_FRAME_DATA_CONTENT_LIGHT_LEVEL),
     ('ICC_PROFILE', lib.AV_FRAME_DATA_ICC_PROFILE),
+    # SEI_UNREGISTERED available since version (56.54.100) of libavutil
```

Comment:
@jlaine sorry for the delay, I missed your comment! This is available in FFmpeg >= 4.4 (more specifically since [this commit](https://github.com/FFmpeg/FFmpeg/commit/bc8ab084fb07419b7cb13b460460b1ae8facf1d1)).

Reply:
Could we have a comment mentioning FFmpeg >= 4.4. please as it's not easy to convert libavutil versions to FFmpeg releases. Having an explicit message will allow us to drop the condition once we only support FFmpeg 4.4 and later.

Reply:
I left the full libavutil version in the comment, but also added the FFmpeg version.